docs/4.4. Behaviour - Access Tokens.md

```diff
@@ -90,24 +90,24 @@ in Section 3.3 of [RFC6749][RFC-6749].
 
 The value of each scope corresponds to the namespace identifier for a given NMOS
 API (e.g. "registration", "query", etc.) to which the user is requesting access and determines how the private
-`x-nmos-api` claim, detailed below, is populated.
+`x-nmos-*` claims, detailed below, are populated.
 
 ## Private Claims
 
-[RFC 7519][RFC-7519] allows for "private claims". The following claim is used to identify the API specification a given
+[RFC 7519][RFC-7519] allows for "private claims". The following claims are used to identify the API specification(s) a given
 token is used for.
 
-### x-nmos-api
+### x-nmos-*
 _Contains information particular to the NMOS API the token is intended for_
 
-The `x-nmos-api` claim MUST be included in the token. The value of the claim is a JSON object, the contents of which
-are `key:value` pairs, with the `key` identifying an individual AMWA NMOS API using the namespace identifier found in
-the URL for that given API (e.g. "registration", "query", etc.), and the `value` being a JSON object indicating access
-permissions for that API. An empty `x-nmos-api` object MUST indicate no access to any of the APIs. A missing API key
-MUST indicate no access to that specific API.
+One or more `x-nmos-*` claims MUST be included in the token. The claim name begins `x-nmos-` followed by the namespace
+identifier found in the URL for that given API (e.g. "registration", "query", etc.).
+
+The value of the claim is a JSON object, indicating access permissions for that API. An empty or missing `x-nmos-*`
+object indicates that no access is permitted to the namespace-identified API.
```

Comment:
> An empty or missing object

I would not expect an empty object as per [here](https://github.com/AMWA-TV/nmos-authorization/pull/39/files#diff-9557aa8e700811ccb1e7c2a0f5ea56c4R118-R120) it should be omitted

Reply:
Good spot.

> If no permissions are given for a specific API, the `x-nmos-*` claim relating to that API key MUST be to ensure token size is minimised.	removed to ensure token size is minimised.

That sentence covers _writers_, while the empty object condition here might currently be necessary for _readers_ due to the schema, but we could deal with that by adding a `"minProperties": 1` constraint?

Reply:
Hopefully the next commit sorts this out.